core-bundle/tests/Security/Authentication/ContaoLoginAuthenticationListenerTest.php

```diff
@@ -126,19 +125,14 @@ public function testStoresLastUsernameInSession(): void
 
     public function testHandlesTwoFactorAuthentication(): void
     {
-        $authenticatedToken = $this->createMock(UsernamePasswordToken::class);
-
         $currentToken = $this->createMock(TwoFactorTokenInterface::class);
-        $currentToken
-            ->expects($this->once())
-            ->method('getAuthenticatedToken')
-            ->willReturn($authenticatedToken)
-        ;
+        $currentTokenWithCredentials = $currentToken->createWithCredentials('123456');
```

Comment:
If you want to use a method of the original class, you have to use `createPartialMock()` IMHO.

Reply:
I've mocked the second token as well in 9d7eb5f66.